src/main/webapp/app/app.scss

```diff
@@ -385,10 +385,10 @@ http://stackoverflow.com/questions/23436430/bootstrap-3-input-group-100-width */
   margin-top: -0.5rem;
 }
 
-.form-control {
-  border: 1px solid #07ade1;
-  padding: 17px;
-}
+// .form-control {
+//   border: 1px solid #07ade1;
+//   padding: 17px;
+// }
```

Comment:
Remove if unused

Reply:
Resolved in dcaf9e6e87fbe5d28af246ab603fa71b1b0dd344